app/experimenter/templates/experiments/edit_overview.html

```diff
@@ -8,7 +8,7 @@
       {{ object.name }}
     </a>
   {% else %}
-    New Experiment
+    New Experiment {{ form.errors }}
```

Comment:
I couldn't get it to display anything. How do I trigger an error to show up?

Reply:
OH GOOD CATCH that was just a debug statement oops!